types/office-js-preview/index.d.ts

```diff
@@ -3140,7 +3140,7 @@ declare namespace Office {
          * @param actionId The ID of an action that is defined in the manifest.
          * @param actionFunction The function that is run when the action is invoked. 
          */
-        associate: (actionId: string, actionFunction: (arg?: any) => void) => void;
+        associate(actionId: string, actionFunction: (arg?: any) => void) => void;
```

Comment:
Does this need to be something like this?

```suggestion
        associate(actionId: string, actionFunction: (arg?: any) => void): Promise<void>;
```

Reply:
> Does this need to be something like this?

It's not async, so I don't think so.

Reply:
right. Updated my suggestion inline without the "Promise"?